python_modules/dagster/dagster/_daemon/utils.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from typing import Optional
+
+from dagster._utils.error import (
+    ExceptionInfo,
+    SerializableErrorInfo,
+    serializable_error_info_from_exc_info,
+)
+
+
+class ErrorCapture:
```

Comment:
should we call this DaemonErrorCapture to avoid confusion with the existing one?

Reply:
I figure that's kind of what our package namespacing is doing for us. But I'm fine with a rename.